### PR TITLE
Add readme file

### PR DESCRIPTION
**Pull request by VLeads**
Added Readme

Done some minor css fixes

### CODE REVIEW

**Review by vrx29 — APPROVED** [review]
readme looks good you can merge it.

**Review by vrinda-mahajan — APPROVED** [review]
Readme looks good.👍

File: components/quizzes/cosmos/cosmos.html [hunk 1]
```diff
@@ -25,7 +25,7 @@
         <div class="logo">
           <a href="#" class="quiz-navbar-text">
             <span class="font-family-kaushan">R</span>apid
-            <span class="font-family-kaushan">Q</span>uiz</a
+            <span class="font-family-kaushan">Q</span>uizz</a
```
Comment (MayankKumar10) [inline components/quizzes/cosmos/cosmos.html:28]: use ::first-letter for different font family

**Review by MayankKumar10 — COMMENTED** [review]
Everything else looks good to me

**Pull request merged**